client/mysqlbinlog.cc

```diff
@@ -969,19 +979,44 @@ static bool print_row_event(PRINT_EVENT_INFO *print_event_info, Log_event *ev,
     */
     if (skip_event)
     {
-      // append END-MARKER(') with delimiter
-      IO_CACHE *const body_cache= &print_event_info->body_cache;
-      if (my_b_tell(body_cache))
-        my_b_printf(body_cache, "'%s\n", print_event_info->delimiter);
-
-      // flush cache
-      if ((copy_event_cache_to_file_and_reinit(&print_event_info->head_cache,
-                                               result_file) ||
-           copy_event_cache_to_file_and_reinit(&print_event_info->body_cache,
-                                               result_file) ||
-           copy_event_cache_to_file_and_reinit(&print_event_info->tail_cache,
-                                               result_file)))
-        return 1;
+      if (!opt_flashback) {
+        // append END-MARKER(') with delimiter
+        IO_CACHE *const body_cache= &print_event_info->body_cache;
+        if (my_b_tell(body_cache))
+          my_b_printf(body_cache, "'%s\n", print_event_info->delimiter);
+
+        // flush cache
+        if ((copy_event_cache_to_file_and_reinit(&print_event_info->head_cache,
+                                                 result_file) ||
+             copy_event_cache_to_file_and_reinit(&print_event_info->body_cache,
+                                                 result_file) ||
+             copy_event_cache_to_file_and_reinit(&print_event_info->tail_cache,
+                                                 result_file)))
+          return 1;
+      }else if(opt_flashback && events_in_stmt.elements > 0){
```

Comment:
```suggestion
      }
      else if (opt_flashback && events_in_stmt.elements > 0)
      {
```

Reply:
Agreed with the good practice which always prevail.
Sure it's a new feature, but let me be pedantic too :smiley: to ask what specifically
do you mean with
>  the behavior of arguments with `,s` 
?
Is it that the OLD would error out in case of the comma separated list?
If something else, could you please come back with an example?
Thank you Jimmy!

Reply:
Yes. OLD: `school,class` outputs no row events given tables cannot have `,` in their names.

The consideration is pedantic because, of course, relying on whether inputs are unsupported to sanitize them is not a secure practice.
What’s broken is an **anti**-feature, not a _feature_.